src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { CardGrid } from '@edx/paragon';
+import { useParams } from 'react-router-dom';
+import ContentHighlightCardItem from './ContentHighlightCardItem';
+import { TEST_COURSE_HIGHLIHTS_DATA } from './data/constants';
+
+const ContentHighlightsCardItemsContainer = () => {
+  const { highlightUUID } = useParams();
+  const [highlightCourses] = useState(
+    TEST_COURSE_HIGHLIHTS_DATA.filter(highlight => highlight.uuid === highlightUUID)[0].courses,
+  );
```

Comment:
Similar to the feedback for the highlights list page, let's stub out a custom hook for the data fetching of the a single highlight, e.g.

```jsx
const [
  contentHighlight,
  isLoadingContentHighlight,
  contentHighlightError,
] = useContentHighlight(highlightUUID);
```

where there's a mock API call that resolves with the `TEST_COURSE_HIGHLIHTS_DATA.filter(...)` similar to the one described in another comment.

We'd probably want this custom hook to ultimately be called once in the `ContentHighlightSet` component so that the data can be shared via React Context to all its subcomponents. For now, though, similar to the other comment, we can call the custom hook twice or pass the necessary data down as props for the time being.

Reply:
Leaving open as a reference to to implement in future PR